py3-meson-python.yaml

```diff
@@ -22,27 +33,51 @@ environment:
       - busybox
       - ca-certificates-bundle
       - meson
-      - py3-colorama
-      - py3-gpep517
-      - py3-packaging
-      - py3-pyproject-metadata
-      - py3-setuptools
-      - py3-tomli
-      - py3-wheel
-      - python-3
+      - py3-supported-packaging
+      - py3-supported-pip
+      - py3-supported-pyproject-metadata
+      - py3-supported-tomli
       - wolfi-base
 
+subpackages:
+  - range: py-versions
+    name: py${{range.key}}-${{vars.pypi-package}}
+    description: ${{vars.pypi-package}}
+    dependencies:
+      provides:
+        - py3-${{vars.pypi-package}}
+      provider-priority: ${{range.value}}
+      runtime:
+        - py${{range.key}}-tomli
+        - py${{range.key}}-packaging
+        - py${{range.key}}-pyproject-metadata
+    pipeline:
+      - uses: py/pip-build-install
+        with:
+          python: python${{range.key}}
+      - uses: strip
+    test:
+      pipeline:
+        - uses: python/import
+          with:
+            python: python${{range.key}}
+            import: mesonpy
+
+  - name: py3-supported-${{vars.pypi-package}}
+    description: meta package providing ${{vars.pypi-package}} for supported python versions.
+    dependencies:
+      runtime:
+        - py3.10-${{vars.pypi-package}}
+        - py3.11-${{vars.pypi-package}}
+        - py3.12-${{vars.pypi-package}}
+
 pipeline:
   - uses: git-checkout
     with:
       repository: https://github.com/mesonbuild/meson-python
       tag: ${{package.version}}
       expected-commit: 4f50bf27ef5282f8147c1df073d27859bda5e890
 
-  - runs: |
-      python3 -m gpep517 build-wheel --wheel-dir dist --output-fd 3 3>&1 >&2
-      python3 -m installer -d "${{targets.destdir}}" dist/*.whl
-
   - uses: strip
 
```

Comment:
Can you
1. move pipeline above subpackages (wierd that 'yam' didn't do that)
2. remove 'uses strip' since... this doesn't do aything at all.